client/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/docker/go-connections v0.6.0
 	github.com/docker/go-units v0.5.0
 	github.com/google/go-cmp v0.7.0
-	github.com/moby/moby/api v1.52.0
+	github.com/moby/moby/api v1.52.1-0.20251217102041-b8093b76fb43
```

Comment:
Guessing this was for testing; can you add back the replace rules before we merge?